### PR TITLE
Hooks Infra and New changes as per the UI requirement

### PR DESCRIPTION
**Pull request by kpunwatk**
Added Updated New Hook Infra for running QE UI hook Tests

### CODE REVIEW

**Review by mturley — APPROVED** [review]
LGTM. I'll wait for @ibragins to review before merging.

**Comment by mturley** [comment]
@kpunwatk I've converted this to a draft for now, when you're ready you can click the "Ready for review" button.

**Comment by kpunwatk** [comment]
> @kpunwatk I've converted this to a draft for now, when you're ready you can click the "Ready for review" button.

@mturley Thank You !

**Comment by kpunwatk** [comment]
@mturley Thank You !

File: qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts:83]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/types/constants.ts
Comment (ibragins) [inline qe-tests/cypress/integration/types/constants.ts:37]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/tests/Rhv/config_separate_mapping_rhv.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/Rhv/config_separate_mapping_rhv.ts:19]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/tests/Rhv/config_separate_mapping_rhv.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/Rhv/config_separate_mapping_rhv.ts:79]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/tests/ocpvirt/ocpVirtConfig.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/ocpvirt/ocpVirtConfig.ts:14]: Please clean all variables that are not in use

File: qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts:19]: env variable name `ansiblePlaybook_1` is not acceptable, please update it to something like `postAnsiblePlaybook`

File: qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts:76]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts
Comment (ibragins) [inline qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts:83]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/types/constants.ts
Comment (ibragins) [inline qe-tests/cypress/integration/types/constants.ts:37]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/types/constants.ts
Comment (ibragins) [inline qe-tests/cypress/integration/types/constants.ts:70]: Please clean all comments of not-required code

File: qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts
Comment (kpunwatk) [reply qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts:83]: Thanks for reviewing, I'll will clean up the not-required code.

**Review by mturley — APPROVED** [review]
LGTM with an initial pass, merging to unblock @ibragins . Will take a closer look after the holiday.

**Pull request merged**
